lineman/app/models/discussion_reader_model.coffee

```diff
@@ -12,11 +12,13 @@ angular.module('loomioApp').factory 'DiscussionReaderModel', (BaseModel) ->
       @volume = null
       @lastReadSequenceId = -1
 
-      if data.discussion_id?
-        discussion = @recordStore.discussions.find(data.discussion_id)
-        _.extend data,
-          id:       discussion.id
-          group_id: discussion.groupId
+      # commented so I can ask gdpelican about it
```

Comment:
@gdpelican I've taken this out because it was giving me problems when inserting my own discussion reader.. I can probably put it back in though.. was just wondering why we do it.. it looks hacky but don't take that personally.